content/quickstart.md

```diff
@@ -39,8 +83,74 @@ Under construction.
 
 ## Loading your model
 
-Under construction.
+#### First time
+
+1- open the ``model`` menu
+
+2- load model
+
+3- open ``.vrm`` file
+
+#### Loading a used model
+
+1- open ``preset`` menu
+
+2- select preset and hit ``load``
+
+## Tracking
+
+### OpenSeeFace
+
+Runs using your webcam on laptop/pc. Source code provided by [emilianavt](https://github.com/emilianavt/OpenSeeFace)
+
+### IFacialMocap
+
+Uses the 3D tracking capabilities of Apple Inc devices to provide optimal tracking data from your phone. Use if your model has
+perfect sync.
+
+iFacialMocap is an app that captures facial movements with an iPhone equipped with FaceID. This app sends the captured motion in real time to 3DCG software such as Maya, Unity, Blender on PC. -IFM developers
+
+
+### MeowFace
+
+An android version of IFacialMocap. Also supports perfect sync models.
+
+MeowFace is a precision face tracking application that can send 42 ARKit-like blendshapes as well as head rotation tracking to PC VTuber applications. -MF developers

```

Comment:
Quote should be encased in double quotes. Should be removed since this could go out-of -date very quickly

"MF" should be MeowFace since the acronym has not been defined any else on the page